packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -438,11 +438,16 @@ test('first', async t => {
     'vault is cleared',
   );
 
-  const liquidations = await E(
-    E(vault).getLiquidationSeat(),
-  ).getCurrentAllocation();
-  t.deepEqual(liquidations.Collateral, AmountMath.make(aethBrand, 566n));
```

Comment:
I expect this amount to show up in the vaultSeat.  `await E(vaultSeat).getPayout('Collateral')`